diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -160,7 +159,6 @@ fun ASTNode.findChildAfter(afterThisNodeType: IElementType, childNodeType: IElem
         }
     }
 
-    log.warn("Not able to find a node with type $childNodeType after $afterThisNodeType")
```

Comment:
Does this mean we can close #141 in this PR?

Reply:
yes, I decided to remove it. You were right - it does not make any sense